frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -9,50 +10,276 @@ Feature: Kiali Istio Config page
+  Scenario: KIA1101 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "foo"
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "danger"
+
+  @crd-validation
+  Scenario: KIA1102 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep"
+    And the VirtualService applies to "sleep" hosts
+    And the VirtualService references "foo" gateways
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "danger"
+
+  @crd-validation
+  Scenario: KIA1104 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep"
+    And the route of the VirtualService has weight 10
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+
+  @crd-validation
+  Scenario: KIA1105 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep" and subset "v1"
+    And the route of the VirtualService has weight 50
+    And the http-route of the VirtualService also has a destination to host "sleep" and subset "v1" with weight 50
+    And a "foo" DestinationRule in the "default" namespace for "sleep" host
+    And the DestinationRule has a "v1" subset for "version=v1" labels
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+
+  @crd-validation
+  Scenario: KIA1106 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep"
+    And the VirtualService applies to "sleep" hosts
+    Given there is a "bar" VirtualService in the "default" namespace with a "bar-route" http-route to host "sleep"
+    And the VirtualService applies to "sleep" hosts
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+    And the "bar" "VirtualService" of the "default" namespace should have a "warning"
+
+  @crd-validation
+  Scenario: KIA1107 validation
+    Given there is a "foo" VirtualService in the "default" namespace with a "foo-route" http-route to host "sleep" and subset "v1"
+    When user selects the "default" namespace
+    Then the "foo" "VirtualService" of the "default" namespace should have a "warning"
+
+# TODO: KIA06xx and KIA07xx does not appear in Istio Config list page. They appear in Svc/workload lists.
+#   Thus, these validations do not belong to this feature file.
+
+# TODO: KIA0801 is only applicable for Maistra. We don't have an environment to run tests for this one.
+
+# TODO: KIA090x no longer apply to current Istio (ServiceRole does not exist anymore)
```

Comment:
In kiali/kiali.io#539 I'm removing the docs of such validations. They are already removed from Kiali's codebase.